packages/cli/src/commands/doctor/healthchecks/androidSDK.ts

```diff
@@ -12,6 +12,10 @@ const getBuildToolsVersion = (): string => {
 
   const buildToolsVersionEntry = 'buildToolsVersion';
 
+  if (!fs.existsSync(gradleBuildFilePath)) {
+    return '';
```

Comment:
Can we return `Not Found` instead? I don't feel too good about returning an empty string, but this way we could be consistent. It also seems like this function can return either non-empty string or `Not Found`. Which makes code on line 52 non-reachable. Can you investigate and remove stuff if possible?